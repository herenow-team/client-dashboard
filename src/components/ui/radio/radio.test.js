import React from 'react'
import renderer from 'react-test-renderer'
import Radio from '.'

describe('<Field /> component', () => {
  it('should render correctly', () => {
    const options = [
      {text: 'Sim', value: true, checked: true},
      {text: 'NÃ£o', value: false, checked: false}
    ]
    const tree = renderer
      .create(<Radio name="something" options={options} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
