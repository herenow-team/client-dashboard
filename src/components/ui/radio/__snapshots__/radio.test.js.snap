// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Field /> component should render correctly 1`] = `
<div
  className="control container"
>
  <label
    className="radio"
  >
    <input
      className="input"
      defaultChecked={true}
      name="something"
      type="radio"
      value={true}
    />
    Sim
  </label>
  <label
    className="radio"
  >
    <input
      className="input"
      defaultChecked={false}
      name="something"
      type="radio"
      value={false}
    />
    NÃ£o
  </label>
</div>
`;
